corehq/apps/settings/static/settings/js/user_api_keys.js

```diff
@@ -0,0 +1,62 @@
+hqDefine("settings/js/user_api_keys", [
+    "jquery",
+    "knockout",
+    'underscore',
+    "hqwebapp/js/initial_page_data",
+    "hqwebapp/js/crud_paginated_list",
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    CRUDPaginatedList
+) {
+
+    var ApiKeyListModel = function () {
+        var self = CRUDPaginatedList.CRUDPaginatedListModel(
+            initialPageData.get('total'),
+            initialPageData.get('limit'),
+            initialPageData.get('page'),
+            {
+                statusCodeText: initialPageData.get('status_codes'),
+                allowItemCreation: initialPageData.get('allow_item_creation'),
+                createItemForm: initialPageData.get('create_item_form'),
+                createItemFormClass: initialPageData.get('create_item_form_class'),
+            }
+        );
+
+        updateApiKey = function (action, apiKeyId) {
```

Comment:
```suggestion
        self.updateApiKey = function (action, apiKeyId) {
```

Reply:
Whoops, I'd meant to define this as a local function